pulp_ansible/app/serializers.py

```diff
@@ -75,6 +77,22 @@ class Meta:
         model = CollectionRemote
 
 
+class CollectionOneShotSerializer(serializers.Serializer):
+    """
+    A serializer for the Collection One Shot Upload API.
+    """
+
+    file = serializers.FileField(
+        help_text=_("The collection tarball."),
+        required=True,
+    )
+
+    sha256 = serializers.CharField(
+        required=False,
+        default=None,
+    )
```

Comment:
What's this field for? Maybe some help text would help?

Reply:
It's an optional sha256 digest of the tarball being submitted, and if submitted will validate. I will add help-text in the next push, that is a good idea.